python/graphscope/client/session.py

```diff
@@ -1089,6 +1094,64 @@ def get_config(self):
         """Get configuration of the session."""
         return self._config_params
 
+    def backup_vineyard_depployment(self, graphIDs: str, path: str, pvc_name: str):
+        if self._cluster_type != types_pb2.K8S:
+            return
+        # check if session exist
+        if self.session_id is None:
+            return
+        vineyard_deployment_name = self._config_params["k8s_vineyard_deployment"]
```

Comment:
Do you need to check if there is a vineyard deployment? The session may be launched with vineyard as a sidecar container.

Reply:
Thanks reminding. I will add a check here.